packages/design-system-web/src/components/geov-entity-props-by-predicate/geov-entity-props-by-predicate.tsx

```diff
@@ -0,0 +1,163 @@
+import { Component, Host, h, Prop, State, EventEmitter, Event } from '@stencil/core';
+import { GeovPaginatorCustomEvent } from '../../components';
+import { sparqlJson } from '../../lib/sparqlJson';
+import { Binding } from '../geov-entity-properties/geov-entity-properties';
```

Comment:
remove this import and create a new interface, matching the bindings of the queries in this file